ui/app/routes/variables.js

```diff
@@ -16,7 +16,7 @@ export default class VariablesRoute extends Route.extend(WithForbiddenState) {
   }
   async model() {
     try {
-      const variables = await this.store.findAll('variable');
+      const variables = await this.store.findAll('variable', { reload: true });
```

Comment:
Because we can now load a partial version of a variable (the filtered relationships of a job), we force reload upon entering the variables route.

Reply:
`praise`:  Great catch! If in the future, we want to optimize for page load times (some pages trigger a ton of waterfall requests), we can use the `shouldBackgroundReloadAll` method in the adapter layer to render what we currently have and fetch the rest of the information in the background.